src/Cache/SchemaCache.php

```diff
@@ -23,6 +23,11 @@ public function __construct(CacheInterface $cache)
         $this->cache = $cache;
     }
 
+    public function clear(): bool
+    {
+        return $this->cache->psr()->clear();
```

Comment:
That would clear all the cache, not schema cache only. It's better for it to remove keys related to cache i.e. use tags.

Reply:
done remove.